app/pages/help/help-page.js

```diff
@@ -1,8 +1,27 @@
-
-exports.onNavigatingTo = function(args) {
-  if (args.isBackNavigation) {
-    return;
+  var frameModule = require("ui/frame");
```

Comment:
Change all `var` to `const` when possible.
only if the variable is going to be reassigned, you can use `let`